README.md

```diff
@@ -1,27 +1,38 @@
-# Before Getting Started
-
-Please be sure when running the front end repository that you have the correct versions of npm and node.
+# Getting Your Environment Set Up: 
+**This project was initialized with python 3.11.3.**
+Please be sure when building the front end repository that you have the correct versions of npm and node.
 The correct version can be found and installed [here](https://nodejs.org/en/download/current).
 
 To check if your versions are correct you can use the following commands:
 Check node version with `node -v` -> This should show v20.2.0 in the console
 Check npm version with `npm -v` -> This should show 9.6.7 in the console
 
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+1.  Clone and change directories to the repository in the directory (ex. Desktop) you would like to work in. `git clone [repo link]`. 
+Development is done on the `dev` branch. Ensure you are working either on `dev` or a branch that has branched off of `dev`. 
+-> To Switch to `dev` or another branch, `git checkout branch_name` ex: `git checkout dev` will switch to branch. 
```

Comment:
Instead of using `->` on the same line, use bullet-points. The steps will be easier to follow, and it'll be easier to add more subsections/bullet-points under each step as needed.
```
1. Clone and change directories to the repository in the directory (ex. Desktop) you would like to work in. `git clone [repo link]`. 
Development is done on the `dev` branch. Ensure you are working either on `dev` or a branch that has branched off of `dev`.
    - To Switch to `dev` or another branch, `git checkout branch_name` ex: `git checkout dev` will switch to branch. 
```

1. Clone and change directories to the repository in the directory (ex. Desktop) you would like to work in. `git clone [repo link]`. 
Development is done on the `dev` branch. Ensure you are working either on `dev` or a branch that has branched off of `dev`.
    - To Switch to `dev` or another branch, `git checkout branch_name` ex: `git checkout dev` will switch to branch.